src/sizzle.js

```diff
@@ -221,6 +221,14 @@ function Sizzle( selector, context, results, seed ) {
 
 	results = results || [];
 
+	 // Balasaheb - this try/catch seems to fix IE 'permission denied' errors as described here:
+         try {
+               document === document; //may cause permission denied
+            }
+        catch (err) {
+                document = window.document; //resets document, and no more permission denied errors.
```

Comment:
See `setDocument`.

Reply:
Below are the issue references.

https://github.com/jquery/jquery/issues/1831

https://stackoverflow.com/questions/18088829/script70-permission-denied-accessing-iframe-in-ie

Reply:
Below are the issue references.

https://github.com/jquery/jquery/issues/1831

https://stackoverflow.com/questions/18088829/script70-permission-denied-accessing-iframe-in-ie